webproject/src/main/webapp/listings.html

```diff
@@ -34,129 +26,104 @@
+        <div class="site-mobile-menu site-navbar-target">
+            <div class="site-mobile-menu-header">
+                <div class="site-mobile-menu-close mt-3">
+                    <span class="icon-close2 js-menu-toggle"></span>
+                </div>
             </div>
+            <div class="site-mobile-menu-body"></div>
+        </div>
 
-            <div class="col-12 col-md-9 main-menu">
-              <nav
-                class="site-navigation position-relative text-right"
-                role="navigation"
-              >
-                <ul
-                  class="site-menu main-menu js-clone-nav mr-auto d-none d-lg-block"
-                >
-                  <li>
-                    <a href="#view-listings" class="nav-link">View Listings</a>
-                  </li>
-                  <li>
-                    <a href="#about-section" class="nav-link">Post a listing</a>
-                  </li>
-                  <li>
-                    <a href="#contact-section" class="nav-link">My Rooms</a>
-                  </li>
-                  <li>
-                    <a href="index.html" class="nav-link">Logout</a>
-                  </li>
-                </ul>
-              </nav>
+        <div class="container d-none d-lg-block">
+            <div class="row">
+                <div class="col-12 text-center mb-4 mt-5">
+                    <h1 class="mb-0 site-logo">
+                        <a href="index.html" class="text-black h2 mb-0">Meap</a>
+                    </h1>
+                </div>
             </div>
+        </div>
+        <header class="site-navbar py-md-4 js-sticky-header site-navbar-target" role="banner">
+            <div class="container">
+                <div class="row align-items-center">
+                    <div class="col-6 col-md-6 col-xl-2 d-block d-lg-none">
+                        <h1 class="mb-0 site-logo">
+                            <a href="index.html" class="text-black h2 mb-0">Meap</a>
+                        </h1>
+                    </div>
+
+                    <div class="col-12 col-md-9 main-menu">
+                        <nav class="site-navigation position-relative text-right" role="navigation">
+                            <ul class="site-menu main-menu js-clone-nav mr-auto d-none d-lg-block">
+                                <li>
+                                    <a href="#view-listings" class="nav-link">View Listings</a>
+                                </li>
+                                <li>
+                                    <a href="#about-section" class="nav-link">Post a listing</a>
+                                </li>
+                                <li>
+                                    <a href="#contact-section" class="nav-link">My Rooms</a>
+                                </li>
+                                <li>
+                                    <a href="index.html" class="nav-link">Logout</a>
+                                </li>
```

Comment:
Same as above; might also have merge conflict here. Will help you resolve it if necessary.

Reply:
okay!